tests/schema/tests.py

```diff
@@ -1894,6 +1894,41 @@ def test_m2m_create_through_custom(self):
     def test_m2m_create_through_inherited(self):
         self._test_m2m_create_through(InheritedManyToManyField)
 
+    def test_m2m_through_remove(self):
+        class LocalAuthorNoteThrough(Model):
+            book = ForeignKey("schema.Author", CASCADE)
+            tag = ForeignKey("schema.LocalAuthorNoteThrough", CASCADE)
```

Comment:
Minor but `self` might be more obvious here?

```suggestion
            tag = ForeignKey("self", CASCADE)
```

If I understand correctly this was done to avoid involving another model/table?

Reply:
Yes.